Sources/Fluent/Schema/Database+Schema.swift

```diff
@@ -30,6 +30,13 @@ extension Database {
         if e.database == nil { e.database = self }
 
         let creator = Schema.Creator(e.entity)
+
+        // add timestamps
+        if E.timestamps {
```

Comment:
reading this here tripped me up for a second, maybe following the convention would actually be better even though we decided on this, `usesTimestamps` is probably going to be better longrun and sticks with some of our other changes like `isClosed` 

(Don't feel strongly, your call 👍 )